package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "private": true,
+  "license": "UNLICENSED",
```

Comment:
Why public domain though? We usually use MIT or Apache 2.0

Reply:
Updated license at commit 1672188, I found that license [here](https://github.com/subspace/subspace/blob/main/node-template-spartan/LICENSE), I supposed to use the same one, but you're right, Jeremiah had also commented on it in a call about only using MIT and Apache 2.0